scorecards-site/static/viewer/index.html

```diff
@@ -850,6 +850,21 @@
                   </a>
                   </div>
               </div>
+              <div class="content"><small class="fw-semibold">API URL:</small>
+                {{#if data.repo.name}}
+                  {{#if data.repo.commit}}
+                    <a class="btn-link" href="https://api.scorecard.dev/{{data.repo.name}}?commit={{data.repo.commit}}" target="_blank" rel="noopener noreferrer">
+                      https://api.scorecard.dev/{{data.repo.name}}
+                    </a>
+                  {{else}}
+                    <a class="btn-link" href="https://api.scorecard.dev/{{data.repo.name}}" target="_blank" rel="noopener noreferrer">
+                      https://api.scorecard.dev/{{data.repo.name}}
```

Comment:
these api links don't work, click through and you'll get 404. you're missing `projects` i think

Reply:
Fixed in [dd058ac](https://github.com/ossf/scorecard-webapp/pull/597/commits/dd058acae1e503354467a78e2e48fb8dbd26b2c9).